pkgs/development/tools/devcontainers-cli/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  nodejs_18
+}:
+let
+  pname = "devcontainers-cli";
+  version = "0.54.1";
+  hash = "sha256-L6sVDmKuFmrf9Bm9M54ABmmWFVB8ZlqU+5gAeITBS1Q=";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchzip {
+    inherit hash;
+    url = "https://registry.npmjs.org/@devcontainers/cli/-/cli-${version}.tgz";
+  };
```

Comment:
We should use buildNpmPackage on the original repo instead of downloading from npm